### DISCUSSION TITLE
Has ScriptKit gotten slower?

### DISCUSSION THREAD

**Discussion by jsjoeio** [johnlindquist/kit#144] [Q&A] [answered]
I'm not sure if it's my computer or ScriptKit but I've recently noticed it takes almost a second longer from global hotkey -> input being received in ScriptKit menu.

Wondering what I can do to troubleshoot and/or fix this.

## Demo

Comparing opening a new tab in browser + typing vs. opening ScriptKit and typing (see keyboard presses near bottom of gif)
![2021-04-14 17 12 30](https://user-images.githubusercontent.com/3806031/114795971-acf4e680-9d44-11eb-8adf-5d634b11fe1b.gif)

## Acitvity Monitor 

![image](https://user-images.githubusercontent.com/3806031/114799367-4ffd2e80-9d4c-11eb-88bd-9181631e778d.png)

**Comment by johnlindquist** [comment, marked as answer]
To answer the title, "No" 😅

Hitting a keyboard shortcut is only showing a window then running whichever script it is mapped to.

Would love to hop on a zoom and try and figure out what's going on though. I'll dm you tomorrow

**Reply by jsjoeio** [reply]
Update: John and I chatted over Twitter DMs.

His solution of clearing the files in the`~/.kenv/cache` seemed to help!
